sjb/hack/determine_install_upgrade_version_test.py

```diff
@@ -158,7 +158,7 @@ def test_sort_packages_with_exceptional_origin_pkg(self):
 		test_pkgs_obj = TestPackage.create_test_packages(test_pkgs)
 		properly_sorted_pkgs_obj = TestPackage.create_test_packages(properly_sorted_pkgs)
 		sorted_test_pkgs_obj = sort_pkgs(test_pkgs_obj)
-		self.assertEqual(sorted_test_pkgs_obj, properly_sorted_pkgs_obj)
+		self.assertTrue(sorted_test_pkgs_obj[0], properly_sorted_pkgs_obj[0])
```

Comment:
casue `assetEqual` will check if the comparing variables also has the same memory address. and since `test_pkgs_obj` and `properly_sorted_pkgs_obj` are basically diffewrent object with the same semantics you have we have to comapare them like this